static/game_operations.js

```diff
@@ -3,7 +3,18 @@ var maxPiles = 5;
 $("document").ready(newGame);
 
 function newGame() {
-  $.getJSON( "/new", updatePage);
+  $.ajax({
+    type:"POST",
+    url: "/new",
+    data: JSON.stringify({
+      "min": $('#min').val(),
+      "max": $('#max').val(),
+      "piles": $('#piles').val()
+    }),
+    contentType: "application/json; charset=utf-8",
+    dataType : 'json',
+    success: updatePage
```

Comment:
should we also have an `error: ` here?

Reply:
We could. I was holding off until we know what we'd want to do in the case of as error.